openmdao/visualization/meta_model_viewer/tests/test_view_mm_cmd_line.py

```diff
@@ -0,0 +1,38 @@
+""" Unit tests for structured metamodels in view_mm. """
+import unittest
+import subprocess
+import os
+
+try:
+    import bokeh
+    from openmdao.visualization.meta_model_viewer.meta_model_visualization import MetaModelVisualization
+except ImportError:
+    bokeh = None
+import openmdao.test_suite.test_examples.meta_model_examples.structured_meta_model_example as example
+
+@unittest.skipUnless(bokeh, "Bokeh is required")
+class ViewMMCommandLineTest(unittest.TestCase):
+
+    def test_unspecified_metamodel(self):
+        script = os.path.join(os.path.dirname(__file__), 'example.py')
+        cmd = 'openmdao view_mm {}'.format(script)
+        output = subprocess.check_output(cmd.split()).decode('utf-8', 'ignore')
+        expected_output = "Metamodel not specified. Try one of the following: ['interp1', 'interp2']."
+        self.assertTrue(expected_output in output)
```

Comment:
follow the check in the assertTrue call with a string explaining the failure.  Otherwise, the person running the tests just sees 'False is not true', which tells them nothing about what went wrong.